canonical-websites/www.ubuntu.com
# remove about/ubuntu-font page

## Done

* remove this page, it isn't linked from anywhere
* remove redirects

## QA

1. go to /about/ubuntu-font
2. see that it 404’s

## Issue / Card

Fixes #1515
